apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -2563,6 +2563,14 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 			.toArray();
 	}
 
+	async countLivechatRoomsByPriority(): Promise<number> {
+		return this.col.countDocuments({ priorityWeight: { $exists: true, $ne: 99 } });
```

Comment:
We don't have an index for priority weight, I think this should be possible as well (the query) by checking on the priorityId prop (for which we already have an index)

can u try?

Reply:
Indeed it is, and we already have this stat  😞  the stat is called `omnichannelRoomsWithPriorities` and you made it! haha

I thought that the `notSpecified`(99) had an associated ID, that's why I didn't go to this path.

Thanks a lot @KevLehman

Reply:
lololol